packages/digitalocean-management/infrastructure/kustomization.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../../../modules/digitalocean/provider
+  - ../../../modules/digitalocean/project
+  - ../../../modules/digitalocean/doks
+
+patches:
+  - path: patch-project.yaml
```

Comment:
The patch files are missing. I guess in the scope of this setup, we don't need to patch anything.

Yet I am not sure how would be expose the cluster secret to ArgoCD with DigitalOccean